ambry-server/src/integration-test/java/com.github.ambry.server/ServerHardDeleteTest.java

```diff
@@ -300,10 +300,13 @@ public void endToEndTestHardDeletes() throws Exception {
     notificationSystem.awaitBlobDeletions(blobIdList.get(4).getID());
 
     time.sleep(TimeUnit.DAYS.toMillis(1));
-    // Changes in this patch: a. New header version has 4 more bytes compared to previous b. BlobProperties increased by 1 byte
-    // c. Encryption Key Record size is 114 for an encryptionKey of size 100. EncryptionKeyRecord could be null if not applicable.
+    // Changes in this patch:
```

Comment:
I think its hard to keep these comments up to date. We should just remove them

Reply:
The only good way to keep this updated may be to start a variable and then keep adding to it with comments describing each addition. That way, it is guaranteed that the comments will be up to date

Reply:
Started a variable and wrote a comment telling folks to comment on how it was changed in the future.